server/views/taskList/caSubmitTask.pug

```diff
@@ -1,23 +1,23 @@
+- var canSendToDm = licenceStatus.stage === 'PROCESSING_CA' || allowedTransitions.caToDm
+- var canSendToRo = licenceStatus.stage === 'ELIGIBILITY' && allowedTransitions.caToRo
```

Comment:
the 'or' looks odd. Surely can't send to DM at any time during final checks?

Reply:
Seems that way 👍